controllers/spacebindingcleanup/spacebinding_cleanup_controller_test.go

```diff
@@ -194,9 +183,20 @@ func TestDeleteSpaceBinding(t *testing.T) {
 	})
 }
 
-func TestDeleteSpaceBindingRequest(t *testing.T) {
+func TestDeleteSpaceBindingRequestWithPublicViewerEnabled(t *testing.T) {
+	testDeleteSpaceBindingRequest(t, true)
+}
+
+func TestDeleteSpaceBindingRequestWithPublicViewerDisabled(t *testing.T) {
+	testDeleteSpaceBindingRequest(t, false)
+}
+
+func testDeleteSpaceBindingRequest(t *testing.T, publicViewerEnabled bool) {
```

Comment:
same as for the previous test, could we just have a sub tests that configures the toolchainconfig differently ?

Reply:
as before, the main scope of this is to test the controller's behaves the same way when the public-viewer is enabled and disabled